Automattic/wp-calypso
# Stats: Removing "stats/show-traffic-highlights" feature flag

#### Proposed Changes

* Removing "stats/show-traffic-highlights" feature flag because the feature is available to everyone

#### Testing Instructions



* open the live branch and navigate to stats pages
* make sure that `Traffic` and `Insights` pages display correctly.
* make sure that `7-day highlight` is visible on top of the `Traffic` page

#### Pre-merge Checklist





Related to #70211
